src/Komiku/KomikuParser.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable linebreak-style */
+import { MangaStreamParser } from "../MangaStreamParser";
```

Comment:
****

Reply:
As you can see, the "Author", "Artists" things won't show. Then custom parser are needed.
Also i need to trim the Titles.